src/components/reusable/tabs/tabs.stories.js

```diff
@@ -69,15 +71,30 @@ export const WithIcons = {
         @on-change=${(e) => action(e.type)(e)}
       >
         <kyn-tab slot="tabs" id="tab1" selected>
-          <kd-icon .icon=${userAvatarIcon}></kd-icon>
+          <kd-icon
+            .icon=${userAvatarIcon}
+            role="img"
+            aria-label="UserAvatar"
+            title="UserAvatar"
```

Comment:
The labels on these icons aren't actually necessary, because the tabs already have visible text. `aria-hidden="true"` would also suffice as an alternative. Can have it either way, just food for thought.

Reply:
@brian-patrick-3 , you mean  icons should not be accessible to screen readers ?

Reply:
As per the discussion, left the icons inaccessible to screen readers and added comment in the task.